web/packages/shared/components/DesktopSession/DesktopSession.tsx

```diff
@@ -362,14 +358,7 @@ export function DesktopSession({
       <TopBar
         isConnected={screenState.state === 'canvas-visible'}
         onDisconnect={() => {
-          setClipboardSharingState(prevState => ({
-            ...prevState,
-            isSharing: false,
-          }));
-          setDirectorySharingState(prevState => ({
-            ...prevState,
-            isSharing: false,
```

Comment:
There's no `isSharing` field in `ClipboardSharingState`/`DirectorySharingState`, so this did nothing.

Reply:
Oh my god I forgot that it's a thing in TypeScript. 🫩